app/code/Magento/Review/Block/Form.php

```diff
@@ -139,7 +139,7 @@ protected function _construct()
             );
         }
 
-        $this->setTemplate('form.phtml');
+        $this->setTemplate('Magento_Review::form.phtml');
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.